pkgs/tools/filesystems/fatresize/parted_nix.patch

```diff
@@ -0,0 +1,83 @@
+diff --git a/aclocal.m4 b/aclocal.m4
+index 18e8176..2e75592 100644
+--- a/aclocal.m4
++++ b/aclocal.m4
+@@ -88,8 +88,15 @@ int main ()
+ 	if ( !(version = ped_get_version ()) )
+ 		exit(1);
+ 	if (sscanf(version, "%d.%d.%d", &major, &minor, &micro) != 3) {
+-		printf("%s, bad version string\n", version);
+-		exit(1);
++		if (sscanf(version, "%d.%d", &major, &minor) != 2)
++		{
++		  printf("%s, bad version string\n", version);
++		  exit(1);
++		}
++		else
++		{
++		  micro = 0;
++		}
+ 	}
+ 	
+ 	if ((major > $parted_config_major_version) ||
+diff --git a/configure b/configure
+index ed31457..a7012b7 100755
+--- a/configure
++++ b/configure
+@@ -4456,7 +4456,7 @@ if test "${ac_cv_lib_parted_ped_device_read+set}" = set; then
+   echo $ECHO_N "(cached) $ECHO_C" >&6
+ else
+   ac_check_lib_save_LIBS=$LIBS
+-LIBS="-lparted  $LIBS"
++LIBS="-lparted $LIBS"
```

Comment:
No, not really - I like things aligned though :)))
Reverting to original :P

2015-08-18 1:10 GMT+03:00 Tobias Geerinckx-Rice <notifications@github.com>:

> In pkgs/tools/filesystems/fatresize/parted_nix.patch
> <https://github.com/NixOS/nixpkgs/pull/9301#discussion_r37243055>:
>
> > ++		{
> > ++		  micro = 0;
> > ++		}
> > + 	}
> > + 	
> > + 	if ((major > $parted_config_major_version) ||
> > +diff --git a/configure b/configure
> > +index ed31457..a7012b7 100755
> > +--- a/configure
> > ++++ b/configure
> > +@@ -4456,7 +4456,7 @@ if test "${ac_cv_lib_parted_ped_device_read+set}" = set; then
> > +   echo $ECHO_N "(cached) $ECHO_C" >&6
> > + else
> > +   ac_check_lib_save_LIBS=$LIBS
> > +-LIBS="-lparted  $LIBS"
> > ++LIBS="-lparted $LIBS"
>
> Is this really needed to fix the build? (I know nothing about autocruft.)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NixOS/nixpkgs/pull/9301/files#r37243055>.
>